### DISCUSSION TITLE
how to changing owner address from multisig address to normal address?

### DISCUSSION THREAD

**Discussion by Reskorey** [filecoin-project/lotus#6925] [General Q&A] [answered]
I changed the owner address fron normal address to multisig address. https://github.com/filecoin-project/lotus/pull/5342   , but I can not chang owner address from multisig address to normal address?   Lotus  version v1.11.0

**Comment by SgtCoin** [comment, marked as answer]
@Reskorey, 

I confirmed that this is not currently possible. I created issue #7665 so that the devs can correct this. Please let me know if I can provide any further assistance.
